### PR TITLE
Final submitted models of NeurIPS2019 challenge

### PR DESCRIPTION
**Pull request by zenghsh3**

### CODE REVIEW

File: examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            'v_tgt_field'][1][5][5]
+
+        target_theta = math.atan2(target_v_z, target_v_x)
+        diff_theta = target_theta
+        res = np.append(res, [diff_theta / np.pi])
+
+        return res
+
+
+def rotate_frame(x, y, theta):
```
Comment (TomorrowIsAnOtherDay) [inline examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py]: It seems that we do not use this function. please remove it.

File: examples/NeurIPS2019-Learn-to-Move-Challenge/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- [osim-rl==3.0.11](https://github.com/stanfordnmbl/osim-rl)
+
+
+## Part1: Final submitted model
+### Test
+- How to Run
+
+  1. Enter the sub-folder `final_submit`
+  2. Download the model file from online stroage service: [Baidu Pan](https://pan.baidu.com/s/12LIPspckCT8-Q5U1QX69Fg) (password: `b5ck`) or [Google Drive](https://drive.google.com/file/d/1jJtOcOVJ6auz3s-TyWgUJvofPXI94yxy/view?usp=sharing) 
+  3. Unpack the file by using:
```
Comment (TomorrowIsAnOtherDay) [inline examples/NeurIPS2019-Learn-to-Move-Challenge/README.md]: Unpack the file:

File: examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+        data = np.load('./official_obs_scaler.npz')
+        self.mean, self.std, self.duplicate_id = data['mean'], data[
+            'std'], data['duplicate_id']
+        self.duplicate_id = self.duplicate_id.astype(np.int32).tolist()
+
+    def _get_observation(self, obs_dict):
+
+        res = []
+
+        # target velocity field (in body frame)
```
Comment (TomorrowIsAnOtherDay) [inline examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py]: remove these commented lines.

File: examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py (hunk 1 above)
Comment (zenghsh3) [reply examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py]: done

File: examples/NeurIPS2019-Learn-to-Move-Challenge/README.md (hunk 2 above)
Comment (zenghsh3) [reply examples/NeurIPS2019-Learn-to-Move-Challenge/README.md]: done

File: examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py (hunk 3 above)
Comment (zenghsh3) [reply examples/NeurIPS2019-Learn-to-Move-Challenge/final_submit/env_wrapper.py]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
Looks good to me.

**Pull request merged**